internal/commands/sbomtest/snykclient/snykclient.go

```diff
@@ -0,0 +1,126 @@
+package snykclient
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net/http"
+)
+
+const sbomTestAPIVersion = "2023-08-31~beta"
+
+type SnykClient struct {
+	http       *http.Client
+	apiBaseURL string
+	orgID      string
+}
+
+func NewSnykClient(httpClient *http.Client, apiBaseURL, orgID string) *SnykClient {
+	return &SnykClient{
+		http:       httpClient,
+		apiBaseURL: apiBaseURL,
+		orgID:      orgID,
+	}
+}
+
+func (s *SnykClient) CreateSBOMTest(ctx context.Context, sbomFileContent []byte) (*SBOMTest, error) {
```

Comment:
as you pointed out already, this should probably take an `io.Reader`, sounds like a great idea.